buildDebug.cmd

```diff
@@ -0,0 +1,16 @@
+@echo off
```

Comment:
It would be nice to have this as a PowerShell script for consistency since we already have a `.ps1` in the repository